Dockerfile

```diff
@@ -7,14 +9,25 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get -y update && \
 
 RUN mkdir -p /app/carto_renderer
 
-ENV LOG_LEVEL INFO
+ENV LOG_LEVEL=INFO
 
-COPY bin/freeze-reqs.sh /app/
 COPY dev-requirements.txt /app/
+
+FROM base AS test
```

Comment:
I removed the .TestDockerfile in favor of this test stage, which will have the benefit of also helping us keep the base image up to date.  This stage is run in the "Test" stage of the Jenkinsfile using `--target test`